ipld-future/README.md

```diff
@@ -0,0 +1,179 @@
+# IPLD Spec v1
+
+Editor: Nicola Greco, MIT
+
+> This specification defines a data model and a naming scheme for linking data with cryptographic hashes.
+>
+> InterPlanetary Linked Data (IPLD) is an information space of inter-linked data, where content addresses and links are expressed using the content's cryptographic hash. IPLD is designed to universally address and link to any data so that the data does not require a central naming authority, the integrity of the data can be verified, and untrusted parties can help distribute the data. This specification describes a data model for structured data and a naming scheme to point to data and subsets of the data. These design goals make it different from earlier data models such as JSON or RDF, and naming schemes such as NI [[RFC6920]](https://tools.ietf.org/html/rfc6920), or Magnet URI.
```

Comment:
> an information space of inter-linked data

Seems very odd phrasing to me. 

> content addresses and links are expressed using the content's cryptographic hash

Which is expressed as a hash? The addresses or the links? Are they the same? Why say links then?